src/containers/LogIn/index.js

```diff
@@ -45,39 +53,41 @@ const Login = () => {
   };
 
   return (
-    <Box containerClassName="login__box">
-      <div className="box__title">
-        NeoPosteos
-      </div>
-      <form onSubmit={handleSubmit(onSubmitHandler)}>
-        <Input
-          placeholder="Email"
-          id="email"
-          type="email"
-          {...register('email', { required: true })}
-        />
-        <Input
-          placeholder="Password"
-          id="password"
-          type="password"
-          {...register('password', { required: true })}
-        />
-        <div className="submit-btn">
-          <Button
-            text="SignUp"
-            type="submit"
-          />
-        </div>
-        <div className="signup-link">
-          <p>
-            Don&apos;t have an account?
-            Shame on you!
-            <br />
-            <Link to="/signup">Sign Up</Link>
-          </p>
+    <div className="login">
+      <Box>
+        <div className="login__title">
+          NeoPosteos
         </div>
-      </form>
-    </Box>
+        <form onSubmit={handleSubmit(onSubmitHandler)}>
+          <Input
+            placeholder="Email"
+            id="email"
+            type="email"
+            {...register('email', { required: true })}
+          />
+          <Input
+            placeholder="Password"
+            id="password"
+            type="password"
+            {...register('password', { required: true })}
+          />
+          <div className="submit-btn">
+            <Button
+              text="Login"
+              type="submit"
+            />
+          </div>
+          <div className="login__link">
+            <p>
+              Don&apos;t have an account?
+              Shame on you!
+              <br />
+              <Link to="/signup">Sign Up</Link>
+            </p>
```

Comment:
<p> by default has a line break after the closing tag so you could do

```suggestion
            <p>
              Don&apos;t have an account?
              Shame on you!
            </p>
            <Link to="/signup">Sign Up</Link>
```